pallets/providers/src/lib.rs

```diff
@@ -812,7 +816,10 @@ pub mod pallet {
         /// - The deposit that the user has to pay to register as a SP is held when the user requests to register as a SP
         /// - If this extrinsic is successful, it will be free for the caller, to incentive state debloating
         #[pallet::call_index(2)]
-        #[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
+        #[pallet::weight({
+			T::WeightInfo::confirm_sign_up_bsp()
+				//.max(T::WeightInfo::confirm_sign_up_msp())
```

Comment:
What happened here with the commented out line?

Reply:
Oops, forgot to uncomment that. Nice catch